### PR TITLE
Development

### PR DESCRIPTION
**Pull request by IBTechRaj**

### CODE REVIEW

**Comment by dipto0321** [comment]
# Good job guys 👏 | You have to work a little bit more 🌁 

The code is clean. Already accomplish the lesson about gradient. Code indentation is good. I like that you reset CSS before applying your custom design. I have pointed below what you need to do next 👇 
1. Please update your readme file. Add contributors name with GitHub profile link and a demo link (you can use rawgithack for this for now.). You should use markdown for styling readme information. Here is the guide (markdown guide)[https://www.markdownguide.org/basic-syntax/].
2. Please see that picture.
 I think this problem happens because of resolution or maybe something wrong with your layout arrange. Just analyze your code. I think you can figure out. Fix that problem. 
3. Found one error in (w3Validator)[https://validator.w3.org/nu/?doc=https%3A%2F%2Fraw.githack.com%2FIBTechRaj%2Fbuilding-with-backgrounds-and-gradients%2Fdevelopment%2Findex.html&showsource=yes&showoutline=yes]. Use again w3validator after fixxing the issue.
4. Please organize your files. Create a folder for css and then put all your css files insidthe e new created folder. After that create another folder call `assets` and then put all those things there. 🐹 

I think other than this. Project is good.

Note: Please send another request for code riview after changing. Let me know if you have any question.

Thanks 🙏

**Comment by DanielMitiku** [comment]
Hey Dipto Karmakar, How do we use gitrawhack? We have made the changes and pushed.

**Comment by dipto0321** [comment]
Hi @IBTechRaj and @DanielMitiku ,

Please check this picture to use [rawgithack](https://raw.githack.com/)


Still, the site problem is not fixed properly. please check this out


Please send another request for code review after fixing this. Let me know if you have any question. Just knock me on slack.
Thanks 🙂

**Review by dipto0321 — COMMENTED** [review]
Please try this:
in your `style.css`
```
.container {
    width: 75%;
    margin: 0 auto;
}
.model {
    display: flex;
}
.model1, .model2, .model3, .model4 {
    /* float: left; */
    /* width: 24%; */
}
.model img {
    width: 100%;
}
```
Please remove `padding-bottom`  or `padding` from [here](https://github.com/IBTechRaj/building-with-backgrounds-and-gradients/blob/f784d03afd883deb31bb141bee020cb3c268bd49/css/assets/style.css#L99-L101). Never use any unit like `px` , `rem` `em` and so on for `0`. Css indentation break , please fix those also. Put `img` folder inside assets also.

File: css/assets/style.css [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+        /*font: 12px/18px "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+*/
+        font-family: "news cycle";
+        color: #fff;
+} 
+a {
+    text-decoration: none;
+}
+
+.container{
```
Comment (dipto0321) [inline css/assets/style.css]: .container {
    width: 75%;
    margin: 0 auto;
}

**Comment by dipto0321** [comment]
# Good job guys 🐹 👏 🎖 

Hi @DanielMitiku and @IBTechRaj ,
All is okay now. So you can merge it into the master 👍 . Best of luck for upcoming projects. Happy Coding!

Thanks 🙏 ☺️

**Pull request merged**